torchgeo/trainers/segmentation.py

```diff
@@ -77,19 +77,28 @@ def __init__(self, **kwargs: Any) -> None:
             in_channels: Number of channels in input image
             num_classes: Number of semantic classes to predict
             loss: Name of the loss function
+            ignore_zeros: Whether to ignore the "0" class value in the loss and metrics
```

Comment:
Why not call it "ignore_index" and make it an int that defaults to zero? That gives the most flexibility.

Reply:
Because nn.CrossEntropyLoss uses -1000 as a None value so there is not an easy way to not ignore.

Reply:
I'm not sure I understand. I'm saying to rename the `ignore_zeros` parameter to `ignore_index` and have it accept an int, not to remove the parameter entirely.

Reply:
I know, but say I don't want to ignore anything -- I can't pass None because nn.CrossEntropyLoss behaves differently than everything else that accepts ignore_index